Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -334,7 +334,6 @@
       - SynthesizerInstrument
       - RPED
       - ClothingShoesBootsMagSci
-      - ClothingShoesBootsMagAdv # Frontier
```

Comment:
No more printable advanced magboots? How come? 🤔 I'm not saying we should or shouldn't keep them, just wondering what the motivation for this change is.

Reply:
I merely removed the second instance of that line, adv magboots are still printable.

Reply:
Oh! I missed that! Good catch. :+1: